tests/CommonExtractorTest.php

```diff
@@ -953,6 +953,43 @@ public function testNoRetryOnCsvError(): void
         }
     }
 
+    public function testSshWithCompression(): void
+    {
+        $this->cleanOutputDirectory();
+        $config = $this->getConfig(self::DRIVER);
+        $config['parameters']['db']['ssh'] = [
+            'enabled' => true,
+            'keys' => [
+                '#private' => $this->getPrivateKey(self::DRIVER),
+                'public' => $this->getEnv(self::DRIVER, 'DB_SSH_KEY_PUBLIC'),
+            ],
+            'sshHost' => 'sshproxy',
+            'localPort' => '33056',
+            'compression' => true,
+        ];
+        $result = ($this->getApp($config))->run();
+        $this->assertExtractedData($this->dataDir . '/escaping.csv', $result['imported'][0]['outputTable']);
+        $this->assertExtractedData($this->dataDir . '/simple.csv', $result['imported'][1]['outputTable']);
+    }
+
+    public function testSshWithFalseCompressionConfigRow(): void
+    {
+        $this->cleanOutputDirectory();
+        $config = $this->getConfigRow(self::DRIVER);
+        $config['parameters']['db']['ssh'] = [
+            'enabled' => true,
+            'keys' => [
+                '#private' => $this->getPrivateKey(self::DRIVER),
+                'public' => $this->getEnv(self::DRIVER, 'DB_SSH_KEY_PUBLIC'),
+            ],
+            'sshHost' => 'sshproxy',
+            'localPort' => '33066',
+            'compression' => true,
```

Comment:
isn't this supposed to be false? Of what does it test? I don't think this needs to be tested separately as disabled compression is the default (i.e. there is already a test that has compression key unset in config).

Reply:
i see the difference now:

        $config = $this->getConfigRow(self::DRIVER);

vs 

        $config = $this->getConfig(self::DRIVER);

It's just it has such weird test method name ;)